net/fetcher.go

```diff
@@ -19,6 +20,7 @@ type Fetcher interface {
 	// this includes the provided `ts`. The TipSet that evaluates to true when
 	// passed to `done` will be in the returned slice. The returns slice of TipSets is in Traversal order.
 	FetchTipSets(ctx context.Context, tsKey types.TipSetKey, from peer.ID, done func(ts types.TipSet) (bool, error)) ([]types.TipSet, error)
+	Notifications() *pubsub.PubSub
```

Comment:
I don't expect to land this with the package `pubsub` as the interface definition. This was the easy thing to do until we decide on a pull or push based system for this information.